testci/test_chipbase.py

```diff
@@ -42,6 +42,42 @@ def test_chips_len(chipbase, chips):
     assert len(chips) == len(chipbase)
 
 
+def test_get_all_chips_by_car_id(chipbase, chips):
```

Comment:
These two methods (`test_get_all_chips_by_car_id`, and `test_get_all_chips_by_cam_id`) repeat **a lot** of code; this should be refactored.